### ISSUE TITLE
Conflict with... deleted instances?

### ISSUE THREAD

**Issue by PascalLadalle** [4ian/GDevelop#4822]
When I pick a level, the scene creates and shows three stars, one star for each difficulty setting, and the stars are greyed out if a difficulty setting hasn't been unlocked yet.
When I click elsewhere, the stars are deleted.
Somehow, I need an action that sets the tint back to white for newly created instances that shouldn't be greyed out.
Am I missing something? 😕 

Dummy example
[My project67.zip](https://github.com/4ian/GDevelop/files/10406832/My.project67.zip)

**Comment by 4ian** [comment]
Indeed, I'm fixing it. The tint of objects being recycled in memory was not reset.

**Comment by PascalLadalle** [comment]
Nice, I didn't even know object recycling was a thing. Greta would be proud. 🙈
